packages/tests/tests/sharding/auto_sharding.spec.ts

```diff
@@ -76,11 +69,7 @@ describe("Autosharding: Running Nodes", function () {
     );
 
     const encoder = createEncoder({
-      contentTopic: ContentTopic,
-      pubsubTopicShardInfo: {
```

Comment:
The whole point of autosharding is to not have to worry about the pubsub topics or shard!